skeleton/webpack/aurelia_project/tasks/run.ext

```diff
@@ -49,7 +49,7 @@ function runWebpack(done) {
   server.listen(opts.port, opts.host, function(err) {
     if (err) throw err;
 
-    reportWebpackReadiness(opts);
+    // reportWebpackReadiness(opts);
```

Comment:
Surely an accident :-)

Reply:
Oh, I missed your commit message.

There are two choices:
1. removing this line like you did, but it does not help existing webpack apps.
2. remove the browser opening in `lib/build/webpack-reporter.js`, this can help all existing apps.

Both ways it can remove some deps (at least npm package "open") from the webapck reporter.

Reply:
Should I revert this change and make changes elsewhere?

Reply:
I think that should work.

Reply:
I changed the code in this [commit](https://github.com/aurelia/cli/pull/1132/commits/1608d1b1c56ffd426da210b1636865936e7fbbcc), so that it fixes existing applications as well.